mcs/class/System.Windows.Forms/System.Windows.Forms/LinkLabel.cs

```diff
@@ -668,9 +668,73 @@ private void CreateLinkPieces ()
 										 PaddingClientRectangle,
 										 string_format);
 
-			for (int i = 0; i < pieces.Length; i ++) {
+            // Get offset for different text alignments
```

Comment:
Nit: TABs vs. spaces, the indentation is broken.

Reply:
Should be fixed now